### DISCUSSION TITLE
PCM5102a no output - Fix

### DISCUSSION THREAD

**Discussion by shrieeiee** [copych/AcidBox#27] [General]
![IMG_20240916_100131 - Copy](https://github.com/user-attachments/assets/d10bddf8-38d1-41d8-a020-e30a77d29ea2)
I built the PCB version of the Acidbox without breadboarding it first because I'm an idiot.  Got no sound and turning on debug showed everything was pretty much in order, so after much anguish and messing around, it turns out these pads needed bridging.

Posted hoping to save someone else's day :)

**Comment by 01GOD** [comment]
Photo was a little ambiguous. I have the other version. So those were sets of 3 pads and 2 were bridged on every set of those?

**Reply by shrieeiee** [reply]
4 sets of 3 pads, board was supplied with them unbridged.  According to https://github.com/pschatzmann/arduino-audio-tools/wiki/External-DAC#hiletgo-pcm5102-i2s-iis-lossless-digital-audio-dac-decoder I've bridged them entirely wrong, but it's working, so...

**Reply by 01GOD** [reply]
Thanks!
